warehouse/admin/flags.py

```diff
@@ -47,6 +49,13 @@ def notifications(self):
             .all()
         )
 
+    def disallow_oidc(self, flag_member=None):
+        global_disallow_oidc = self.enabled(AdminFlagValue.DISALLOW_OIDC)
+        if flag_member is None:
+            return global_disallow_oidc
+
+        return self.enabled(flag_member) or global_disallow_oidc
```

Comment:
This is fairly confusing, lots of double-negatives here. I think this could become:

```suggestion
    def disallow_oidc(self, flag_member=None):
        return self.enabled(flag_member) or self.enabled(AdminFlagValue.DISALLOW_OIDC)
```

we'd need to make the following change to `enabled`:

```diff
    def enabled(self, flag_member):
-       flag = self.request.db.get(AdminFlag, flag_member.value)
+       flag = self.request.db.get(AdminFlag, flag_member.value) if flag_member else None
        return flag.enabled if flag else False
```

Reply:
That's much cleaner, thanks!

Reply:
7dce63c3e..5a56bc025